### PR TITLE
Componentise and Enhance User Profile Page

### PR DESCRIPTION
**Pull request by NipuniBhagya**
**Overview of User Profile Page**
image

**Edit section view**
image

**Associated Accounts Section**
image

**Success notification**
image

### CODE REVIEW

File: apps/user-portal/src/components/settings-section.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+/**
+ * Proptypes for the settings section component.
+ */
+interface ComponentProps {
+    header: string;
+    description: string;
+    actionTitle: string;
+    isEdit: boolean;
+    onClick: any;
```
Comment (jeradrutnam) [inline apps/user-portal/src/components/settings-section.tsx]: Make it a function type instead `any`

File: apps/user-portal/src/components/edit-section.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+/**
+ * Edit section component.
+ *
+ * @param {React.PropsWithChildren<any>} props
+ * @return {any}
+ */
+export const EditSection: React.FunctionComponent<any> = (props): JSX.Element => {
+    return (
+        <Segment padded style={{ background: "#f9fafb", border: "none" }}>
```
Comment (jeradrutnam) [inline apps/user-portal/src/components/edit-section.tsx]: Remove inline styles

File: apps/user-portal/src/components/settings-section.tsx (hunk 1 above)
Comment (NipuniBhagya) [reply apps/user-portal/src/components/settings-section.tsx]: Resolved with https://github.com/wso2/identity-apps/pull/58/commits/07ca92c8bee410cb2df318d1010aef36700012a7

File: apps/user-portal/src/components/edit-section.tsx (hunk 2 above)
Comment (NipuniBhagya) [reply apps/user-portal/src/components/edit-section.tsx]: Resolved with https://github.com/wso2/identity-apps/pull/58/commits/07ca92c8bee410cb2df318d1010aef36700012a7

**Comment by NipuniBhagya** [comment]
Please review and merge the updated PR https://github.com/wso2/identity-apps/pull/58

**Pull request closed without merging**